scripts/jest/config.base.js

```diff
@@ -20,10 +20,20 @@ module.exports = {
   rootDir: process.cwd(),
   roots: ['<rootDir>/packages', '<rootDir>/scripts'],
   collectCoverageFrom: ['packages/**/*.js'],
-  timers: 'fake',
+  // TODO: Use the default `modern` implementation (includes `jest.useFakeTimers()`)
+  fakeTimers: {
+    enableGlobally: true,
+    legacyFakeTimers: true,
+  },
+  snapshotFormat: {
+    escapeString: true,
+    printBasicPrototype: true,
```

Comment:
Probably want to follow-up, use the new defaults and see if you like the snapshots better.